src/MgvOfferTaking.sol

```diff
@@ -524,53 +570,58 @@ abstract contract MgvOfferTaking is MgvHasOffers {
           /* This code must be unreachable except if the call to flashloan went OOG and there is enough gas to revert here. **Danger**: if a well-crafted offer/maker offerList can force a revert of `flashloan`, Mangrove will be stuck. */
           revert("mgv/swapError");
         }
       }
 
-      /* Delete the offer. The last argument indicates whether the offer should be stripped of its provision (yes if execution failed, no otherwise). We cannot partially strip an offer provision (for instance, remove only the penalty from a failing offer and leave the rest) since the provision associated with an offer is always deduced from the (gasprice,gasbase,gasreq) parameters and not stored independently. We delete offers whether the amount remaining on offer is > density or not for the sake of uniformity (code is much simpler). We also expect ratios to move often enough that the maker will want to update their ratio anyway. To simulate leaving the remaining volume in the offer, the maker can program their `makerPosthook` to `updateOffer` and put the remaining volume back in. */
-      dirtyDeleteOffer(offerList.offerData[sor.offerId], sor.offer, sor.offerDetail, mgvData != "mgv/tradeSuccess");
+      /* Delete the offer. The last argument indicates whether the offer should be stripped of its provision (yes if execution failed, no otherwise). We cannot partially strip an offer provision (for instance, remove only the penalty from a failing offer and leave the rest) since the provision associated with an offer is always deduced from the (gasprice,gasbase,gasreq) parameters and not stored independently. We delete offers whether the amount remaining on offer is > density or not for the sake of uniformity (code is much simpler). We also expect prices to move often enough that the maker will want to update their offer anyway. To simulate leaving the remaining volume in the offer, the maker can program their `makerPosthook` to `updateOffer` and put the remaining volume back in. */
+      dirtyDeleteOffer(
+        offerList.offerData[sor.offerId], sor.offer, sor.offerDetail, internalMgvData != "mgv/tradeSuccess"
+      );
     }
   }
 
   /* ## Flashloan */
   /* Externally called by `execute`, flashloan lends money to the maker then calls `makerExecute` to run the maker liquidity fetching code. If `makerExecute` is unsuccessful, `flashloan` reverts (but the larger orderbook traversal will continue). 
 
   In detail:
-  1. Flashloans `takerGives` `inbound_tkn` from the taker to the maker and returns false if the loan fails.
+  1. Flashloans `takerGives` units of `sor.olKey.inbound` from the taker to the maker and returns false if the loan fails.
   2. Runs `offerDetail.maker`'s `execute` function.
-  3. Returns the result of the operations, with optional makerData to help the maker debug.
+  3. Returns the result of the operations, with optional `makerData` to help the maker debug.
 
   Made virtual so tests can instrument the function.
   */
```

Comment:
"/* Note : as a possible future feature, we"... delete that section? Since we actually argued that stopping at gasleft is an issue due to gas estimation methodology.